gitlab_issues_mcp.py

```
@@ -0,0 +1,332 @@
+    try:
+        response_clean = response.strip()
+        if response_clean.startswith("```json"):
+            response_clean = response_clean[7:]
+        if response_clean.startswith("```"):
+            response_clean = response_clean[3:]
+        if response_clean.endswith("```"):
+            response_clean = response_clean[:-3]
+        
+        issue_data = json.loads(response_clean.strip())
+        return issue_data["title"], issue_data["description"]
+    except (json.JSONDecodeError, KeyError) as e:
```

Ruff (0.15.2)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)